packages/qualys_vmdr/data_stream/asset_host_detection/elasticsearch/ingest_pipeline/default.yml

```
@@ -1066,6 +1142,70 @@ processors:
       tag: rename_qualys_vmdr_asset_host_detection_vulnerability_RESULTS_1
       target_field: qualys_vmdr.asset_host_detection.vulnerability.results
       ignore_missing: true
+  # The below 2 gsub processors facilitate `splitOnToken` in subsequent script processor which otherwise cannot split \n or \t.
```

```suggestion
  # The next two gsub processors facilitate `splitOnToken` in the subsequent script processor, which otherwise cannot split \n or \t.
```
Do we know that `;` and `|` do not otherwise appear in the text?

---

I checked most values and couldn't find them, hence I chose these delimiters. However to be safe, I changed it to 2 delimiters in  https://github.com/elastic/integrations/pull/11999/commits/7d316538c752e8e035f59abb592f78a1e76de14f.